packages/stencil/src/components/datacom-avataravatar/datacom-avataravatar.css

```diff
@@ -0,0 +1,43 @@
+:host {
+  display: block;
+}
+
+.avatar {
```

Comment:
I believe we should be prefixing our CSS files with 'dc'. So this class (and all others) should be something like '.dc-avatar' rather than just '.avatar'.

Reply:
@AngeCast We can be a bit flexible on this for shadow components as the css is entirely private to the component in the shadow dom.

Reply:
I guess I'm suggesting more from a convention standpoint, as a number of our components use the 'dc-' prefix. Completely fine if we're saying that prefix isn't important though, as long as we're consistent.